kornia/augmentation/augmentation.py

```diff
@@ -1988,8 +2037,9 @@ def __init__(
         return_transform: bool = False,
         same_on_batch: bool = False,
         p: float = 0.5,
+        keepdim: bool = False
     ) -> None:
-        super().__init__(p=p, return_transform=return_transform, same_on_batch=same_on_batch, p_batch=1.0)
+        super().__init__(p=p, return_transform=return_transform, same_on_batch=same_on_batch, p_batch=1.0, keepdim=False)
```

Comment:
Why always False?

Reply:
Oups it's a typo, should be `keepdim=keepdim`, I will re-check everything tomorrow if you want.